src/scss/constants.scss

```diff
@@ -67,6 +69,7 @@ $footer-height: 30px;
 $chat-input-height: 114px;
 $announcement-banner-height: 50px;
 $admin-spaces-sidebar-width: 20%;
+$admin-navbar-height: 36px;
```

Comment:
NOTE: Reusable constant to help UI uniformity across the admin. Ideally all admin tabs should use it to calculate their remaining screen height.